docs/data/introduction/Datalad.md

```diff
@@ -148,9 +149,9 @@ shred -vxzn0 **/T1w?*
 datalad save **/T1w?*
 ```
 
-More realistically, in a job script, rather than including pattern matching such has `**/T1w?*` for every single command expected to modify a dataset file, we'll want to use some command to produce a file list we want to operate on, and store that list in a variable. This is important for readability, but also because actions taken between the `datalad unlock` and the `datalad save` commands might cause the `**/T1w?*` filename pattern to match *a different set files depending on when it runs*.
+More realistically, in a job script, rather than including pattern matching such has `**/T1w?*` for every single command expected to modify a dataset file, we'll want to use some command to produce a file list we want to operate on, and store that list in a variable. This is important for readability, but also because actions taken between the `datalad unlock` and the `datalad save` commands might cause the `**/T1w?*` filename pattern to match *a different set of files depending on when it runs*.
 
-That is to say, because this is stateful, state management is important. If your command might *create* new files, make sure you pop a new variable that includes the newly created filenames; likewise if your job *deletes* unlocked files, make sure your variable *dis*includes files that won't be there after your job is done. If files are renamed, make sure you unlock them under the old name and save them under the new name. Using a matching pattern like `**/T1w?*` each time provides no guarantees of correct behaviour.
+That is to say, because this is stateful, state management is important. If your command might *create* new files, make sure your script makes a new variable that includes the newly created filenames; likewise if your job *deletes* unlocked files, make sure your variable *dis*includes files that won't be there after your job is done. If files are renamed, make sure you unlock them under the old name and save them under the new name. Using a matching pattern like `**/T1w?*` each time provides no guarantees of correct behaviour, if you will be changing the number or name of files in the dataset.
```

Comment:
Suggest replacing mention of 'state management' with something a little more explicit like 'When using patterns to match filenames, we need to be careful of accidentally saving unwanted data created during processing.' 

Maybe eventually we could add in a worked example (dream big)